bazel_example/BUILD.bazel

```diff
@@ -19,14 +20,27 @@ csharp_grpc_library(
     srcs = [
```

Comment:
Pleaes rename this target name to `"example_csharp_grpc"` according to bazel conventions

Reply:
done